tests/unit/test_umm_var.py

```diff
@@ -218,7 +218,7 @@ def test_get_umm_var_absent_fields_removed(self):
         # values have been removed from the record):
         self.assertSetEqual(
             set(umm_var_record.keys()),
-            {'Name', 'LongName', 'Definition', 'DataType',
+            {'Name', 'LongName', 'Definition', 'DataType', 'VariableType',
```

Comment:
I think this test should also update the assertion below this one that checks that the UMM-Var JSON record contains a subset matching the expected record. In this case, `precipitationCal` should also have `'VariableType': 'SCIENCE_VARIABLE'`.

Reply:
* [ ] Add `'VariableType': 'SCIENCE_VARIABLE'`

Reply:
FYI this variable `/precipitationUncal` is not a science variable. So I made a different test for a known science variable. I will also make another test to make sure dimension variables aren't getting marked as science variables.

Reply:
I updated `test_get_umm_var_all_fields` instead since that test has dimension variables assigned to science variables